semopenalex-app/data/templates/Template%3Ahttp%3A%2F%2Fwww.w3.org%2F2004%2F02%2Fskos%2Fcore%23Concept.html

```diff
@@ -0,0 +1,506 @@
+  }
+  .soa .table td {
+    height: auto;
+    padding: 0.5rem 0;
+  }
+  .soa .search-concepts .DataTable--table-grid-layout {
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 0.5rem;
+  }
+  .soa .search-concepts .broad .DataTable--table-grid-layout, 
+  .soa .search-concepts .broader .DataTable--table-grid-layout,
+  .soa .search-concepts .broad,
+  .soa .search-concepts .broader {
+    align-items: flex-end;
+  }
+  .search-concepts .card__title {
+    padding-bottom: 0;
+  }
+  .search-concepts .DataTable--header-container {
+    padding-top: 0;
+  }
+  .search-concepts .DataTable--footer {
+    margin-top: 1.2rem;
+  }
+  .search-concepts .page-item .page-link {
+    background: transparent;
+    border: 1px solid #F3F1EE;
+  }
+  .search-concepts .page-item.active .page-link {
+    color: #9F9F9F;
+  }
+  .concept {
+    color: var(--soa-color-blue);
+    cursor: pointer;
+    padding: 2px 6px;
+    font-size: 1.2rem;
+  }
+  .concept.selected {
+    background: #ECF0F6;
+    border-radius: 0.5rem;
+  }
+  .narrower .concept, .broader .concept {
+    cursor: initial;
+  }
+  .narrower, .broader {
+    padding-top: 5.6rem;
+  }
+  .warning {
+    display: flex;
+    background: rgba(1, 1, 1, 0.05);
+    padding: 0.5rem;
+    border-radius: 0.5rem;
+    text-align: center;
+    justify-content: center;
+    font-size: 1rem;
+    color: var(--soa-color-brown-darker);
+  }
+  .top-authors ul {
+    padding: 0;
+    list-style: none;
+  }
+  .top-authors li {
+    display: flex;
+    align-items: center;
+    gap: 0.8rem;
+  }
+  .co-author__icon {
+    background-image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTIiIGhlaWdodD0iMTIiIHZpZXdCb3g9IjAgMCAxMiAxMiIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTYgNkM3LjY1NzUgNiA5IDQuNjU3NSA5IDNDOSAxLjM0MjUgNy42NTc1IDAgNiAwQzQuMzQyNSAwIDMgMS4zNDI1IDMgM0MzIDQuNjU3NSA0LjM0MjUgNiA2IDZaTTYgNy41QzMuOTk3NSA3LjUgMCA4LjUwNSAwIDEwLjVWMTEuMjVDMCAxMS42NjI1IDAuMzM3NSAxMiAwLjc1IDEySDExLjI1QzExLjY2MjUgMTIgMTIgMTEuNjYyNSAxMiAxMS4yNVYxMC41QzEyIDguNTA1IDguMDAyNSA3LjUgNiA3LjVaIiBmaWxsPSIjMjA1QkFDIi8+Cjwvc3ZnPgo=);
+    background-repeat: no-repeat no-repeat;
+    background-position: center center;
+    background-size: cover;
+    width: 12px;
+    height: 12px;
+  }
+  .publications-since__label, .publications-count__label {
+    font-size: 1.3rem;
+  }
+  .publications-since__value, .publications-count__value {
+    font-size: 3.2rem;
+    font-weight: 700;
+  }
+  .publications-since__label, .publications-since__value {
+    color: #000000;
+  }
+  .publications-count__label, .publications-count__value {
+    color: var(--soa-color-green);
+  }
+</style>
+<div class="soa">
+  <div class="header">
+    <mp-label iri='{{page-resource}}' class="header__title-main"></mp-label>
+  </div>
+  <div class="row cards">
+    <div class="card publication-chart">
+      <div>
+        <div>
+          <div class="publications-since__label">Publications since</div>
```

Comment:
this info is extremely expensive to query especially if publications has more than 100 millions and needed to sort publication year. This kills DB several times. Therefore, i propose to switch to show `Publications` count. 
You can get this figure using `worksCount`.